Tests/SentryTests/Helper/SentrySerializationTests.swift

```diff
@@ -245,6 +245,23 @@ class SentrySerializationTests: XCTestCase {
         XCTAssertNil(actual)
     }
 
+    func testDictionaryToBaggageEncoded() {
```

Comment:
`m`: Can we add a test for checking if the Baggage exceeds the max size please?

Reply:
We already have it. 
https://github.com/getsentry/sentry-cocoa/blob/76a71667fda5c54fda1ac82dbb99607e7a52edb5/Tests/SentryTests/Helper/SentrySerializationTests.swift#L258-L262